content/post/perfect-analytics-org-structure.md

```diff
@@ -0,0 +1,54 @@
+
+## Embedded
+
+The next step are embedded Analytics functions in which there are designated Analysts in each team, that is people whose job title actually is something like *Analyst*, grouped together in each business unit. There might be an Analytics team for the Sales organization, one for Marketing, and another one for IT Operations.
+
+This is a big advancement. Analysts can exchange ideas with each other, align on methodologies and become develop common standards for the unit they're working in.
+
+While this is a common setup, it usually is a symptom of a siloed company culture in general where different business function work independently from each other and any alignment between functions can only be cascaded down from the top management levels.
+
+It also comes with the downside that ressources cannot easily be shifted between units.
+
+## Centralized
+
+Let's break up those silos then! Merge all Analysts into a central group with a standardized intake process. The Analytics team then allocates ressources to each "ticket" and fulfills the requests.
+
+While this can be the most ressource-effective approach, it's often *politically* difficult because it means that different business units compete about the same Analysts. It requires either a strong Analytics lead with a clear and trusted voice in setting priorities, or a system of cost-allocation whereby Analyst's time is allocated to different projects. Companies who have project-based time tracking anyway (agencies, law offices) are a natural fit for this.
+
+## Central-Embedded
+
+So what should we do then? Those are all the options, no?
+
+This brings us to option number four. Basically a mix of the two previous organizational setups, the *central-embedded* org structure consists of an Analytics department which consists of all Analysts in the company. Nevertheless, those Analysts are also associated with different business units, meaning that every Analyst has a direct reporting line to his Analytics manager and a dotted line to the manager of the team they're supporting.
+
+This allows the Analytics function to work as one team and set common standards and guidelines. It also provides a common platform to resolve difficulties. Resources can be allocated more flexibly than in a pure embedded function; not only depending on the current needs of the company, but also based on each Analyst's career objectives.
+
+At the same time, each Analyst can become a domain expert and deeply connected ressource for the team they're supporting. They bridge the communication within the team with cross-functional alignment throughout the Analytics team.
+
+There can be prioritization struggles between the different group managers and the Analytics manager. Not only a challenge, this can also be a great opportunity because it requires different department heads to be in close exchange and alignment with each other.
```

Comment:
What are some ways to ensure alignment between department heads? Are there strategies to employ when head of Marketing wants the Marketing Analyst to work on attribution modeling but the head of Data wants the Analyst to focus on internal ETL refactoring?